### ISSUE TITLE
HIGH: Shared VT's

### ISSUE THREAD

**Issue by AaronJowitt** [mairlist/rebelreq2018#1]
When VT asset ID's are externally scheduled (MM), when a given VT ID is hared across multiple playlists, the original VT pre/post event metadata/segue points (Assuming sequence of events are the same/similar) should be retained across all playlists
--

**Comment by tweibert** [comment]
I wonder how this will work when the same VT is used with different music items - the segues of a particular VT usually depend on the music items, don't they?

**Comment by AaronJowitt** [comment]
Yes, exactly right.  We always retain the same music with VT sequence.  

StationA; MUS/T665 > VTR/54P > MUS/R458
StationB; MUS/T665 > VTR/54P > MUS/R458

We never change the Pre-VT item, and when we have a different post VT item, then teh VT is recorded with a EOM/Start next marker right at the end of the last word.  This is typically for going into commercial breaks that are different on stations that share the same music log, and share many (but not all) VT's.  i.e. ;

StationA; MUS/T665 > VTR/54P > COM/A568
StationB; MUS/T665 > VTR/54P > COM/B999

WO has a link protect feature (which is really nice, but I stress we can live without link protect), where when you record a VT, there is a check box (link Pre VT) next to the Pre VT event, and a check box next to the post VT event.  If you tick the box (and we can even set the status of each box with flags in the MM export - cool), then WO locks that pre VT event to the VT.  If you 'break the link' by changing the pre VT item, then WO will refuse to air the VT and skips it.  In our case, we're just have a staff rule you can't chnage the pre VT event without re-recording the VT.  Gets us by.
